<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络聊天室</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="status_label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>您进入了聊天室</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Image_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;
border-radius: 10px;
background-color: rgb(79, 132, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>图片</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTextEdit" name="input_text">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>8</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="send_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>10</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(68, 44, 255);
border: 0px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Return</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTextBrowser" name="info_text">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>会员服务</string>
    </property>
    <addaction name="actionVIP"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="Change_Name_Button"/>
    <addaction name="Exit_Login"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_To_Server">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionasdfadsf">
   <property name="text">
    <string>asdfadsf</string>
   </property>
  </action>
  <action name="actionCreate_User">
   <property name="text">
    <string>注册新用户</string>
   </property>
  </action>
  <action name="actionVIP">
   <property name="text">
    <string>充值VIP会员</string>
   </property>
  </action>
  <action name="actionasdfadsf_2">
   <property name="text">
    <string>asdfadsf</string>
   </property>
  </action>
  <action name="Change_Name_Button">
   <property name="text">
    <string>修改昵称</string>
   </property>
  </action>
  <action name="Exit_Login">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
